### DISCUSSION TITLE
Vehicle SoC: Mercedes me stopps BYOCAR by August 31, 2023

### DISCUSSION THREAD

**Discussion by sigma415** [camueller/SmartApplianceEnabler#551] [Allgemein]
Please be aware, that Mercedes me Developer stopps BYOCAR by August 31, 2023 for private customers.
See here: https://developer.mercedes-benz.com/news/bringyourowncar-sundown

At this moment no alternative is available and I also got the information from MB me developer support, that there is no timeplan for the promised new solution available.

See also discussions here (in German):
https://github.com/evcc-io/evcc/discussions/9166
https://github.com/evcc-io/evcc/discussions/9452

**Comment by solanjo** [comment]
Mit Hilfe einer iobroker-Installation und den Adaptern mercedesme sowie Simple-API ist es möglich, den benötigten SoC dem SAE zur Verfügung zu stellen.
Andere User berichten auch von einem Mercedes-Plugin für Homeassistant.
